collab-service/src/message_handler/internal/internal_message_types.d.ts

```diff
@@ -0,0 +1,25 @@
+const enum ConnectionFlag {
+  NONE,
+  JOIN,
+  ACK,
```

Comment:
Rather than doing this, an OPCode approach would be neater. `NONE` would translate to a `DATA` op instead.